index.html

```diff
@@ -10,129 +10,129 @@
-      </div>
-      <div class="heading-2">
+        <ul class="column">CANOPY
+          <li><a href="#"><img src="images/point.jpg"></a></li>
+          <li><a href="#"><img src="images/back.jpg"></a></li>
+          <li><a href="#"><img src="images/point.jpg"></a></li>
+          <li><a href="#"><img src="images/year.jpg"></a></li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="works">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul> 
+        <nav>
+          <button type="button" class="button">See Project</button>
+        </nav>
+      </section>
+  
+      <section class="page-5">
+        <img class="snapshoot" id="snapshoot-R" src="images/Snapshoot Portfolio 4.jpg">
         <h2>Multi-Post <br> Stories</h2>
-      </div>
-      <ul class="column">CANOPY
-        <li><a href="#"><img src="images/point.jpg"></a></li>
-        <li><a href="#"><img src="images/back.jpg"></a></li>
-        <li><a href="#"><img src="images/point.jpg"></a></li>
-        <li><a href="#"><img src="images/year.jpg"></a></li>
-      </ul>
-      <p>A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</p>
-      <nav class="work">
-        <a href="#"><img src="images/Tags.jpg"></a>
-      </nav>
-      <nav>
-        <button type="button" class="button">See Project</button>
-      </nav>
-    </div>
-
-    <div class="about-myself">
-      <div class="heading-3">
-        <h2>About <br> Myself</h2>
-      </div> 
-      <p>Hello I'm a software developer! I can help <br> you build a product , feature or website <br> Look through some of my work and <br> experience! If you like what you see and <br> have a project you need coded, don't <br> hestiate to contact me.</p>
-      <a class="links">LET'S CONNECT</a>
+        <ul class="column">CANOPY
+          <li><a href="#"><img src="images/point.jpg"></a></li>
+          <li><a href="#"><img src="images/back.jpg"></a></li>
+          <li><a href="#"><img src="images/point.jpg"></a></li>
+          <li><a href="#"><img src="images/year.jpg"></a></li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="works">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul> 
+        <nav>
+          <button type="button" class="button">See Project</button>
+        </nav>
+      </section>
+    </section>
+  
+    <section class="about-myself">
+      <div class="about-myself-2"><h2>About <br> Myself</h2>
+        <p>Hello I'm a software developer! I can help <br> you build a product , feature or website <br> Look through some of my work and <br> experience! If you like what you see and <br> have a project you need coded, don't <br> hestiate to contact me.</p>
+        <a class="links">LET'S CONNECT</a>
       <div>
```

Comment:
- [ ] Good job @AyomipoMajek, Yet as you can notice here your design is a bit off compared with the Figma design. one of the main goals of this exercise is to make you capable of parsing a design into the exact layout. Let's fix this to make your design even better.

## Your design
![your-about](https://user-images.githubusercontent.com/69077061/201469106-fe1376a7-911f-4fc1-b2c2-c26568641703.PNG)

### Figma design.
![figma-about](https://user-images.githubusercontent.com/69077061/201469104-53fc6944-c155-4feb-ae77-77995e15e52b.PNG)